internal/provider/resource_autogen_subdomain.go

```diff
@@ -147,12 +147,11 @@ func (r *resourceAutogenSubdomain) Create(ctx context.Context, planned map[strin
 			Detail:   err.Error(),
 		})
 	} else if autogenSubdomain == nil {
-		state["id"] = tftypes.NewValue(tftypes.String, "")
-		state["subdomain_id"] = tftypes.NewValue(tftypes.Number, &subdomainId)
-		state["env"] = tftypes.NewValue(tftypes.String, envName)
-		state["dns_name"] = tftypes.NewValue(tftypes.String, "")
-		state["domain_name"] = tftypes.NewValue(tftypes.String, "")
-		state["fqdn"] = tftypes.NewValue(tftypes.String, "")
+		diags = append(diags, &tfprotov5.Diagnostic{
```

Comment:
This isn't always true, it could fail to find the autogen_subdomain even with subdomain and env being present.